### PR TITLE
Fix #78 + #79

### PR DESCRIPTION
**Pull request by z3cka**
fixes for #78 & #79 when you apply them together. @emb03 please review.

### CODE REVIEW

File: themes/bootstrapndlc/templates/sessions.html.twig [hunk 1]
```diff
@@ -1,32 +1,15 @@
...
+  <ul>
+    {% for child in collection %}
+      <li><h2>{{ child.title }}</h2></li>
+      <ul>
+        <li><strong>Sumary</strong>: <p>{{ child.summary }}</p></li>
+        <li><strong>Format</strong>: {{ child.header.taxonomy.format }}</li>
+        <li><strong>Track</strong>: {{ child.header.taxonomy.track }}</li>
+      </ul>
+    {% endfor %}
+  </ul>
```
Comment (emb03) [inline themes/bootstrapndlc/templates/sessions.html.twig]: What happened to the sidebar menu?

Comment (z3cka) [inline themes/bootstrapndlc/templates/sessions.html.twig]: nothing. that's an outdated diff...

Comment (z3cka) [inline themes/bootstrapndlc/templates/sessions.html.twig]: this PR doesn't touch `themes/bootstrapndlc/templates/sessions.html.twig`

File: pages/00.sessions/02.subjective-subject-headings/default.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (emb03) [inline pages/00.sessions/02.subjective-subject-headings/default.md:1]: why does it say default.md? everything in sessions should be called session.md

File: pages/00.sessions/sessions.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (emb03) [inline pages/00.sessions/sessions.md:1]: don't recognize this file at all

Comment (emb03) [inline pages/00.sessions/sessions.md:1]: where is all the rest of the stuff?

File: pages/00.sessions/02.subjective-subject-headings/default.md (hunk 2 above)
Comment (z3cka) [inline pages/00.sessions/02.subjective-subject-headings/default.md:1]: sample files... will delete

File: pages/00.sessions/sessions.md (hunk 3 above)
Comment (z3cka) [inline pages/00.sessions/sessions.md:1]: in #78

Comment (z3cka) [inline pages/00.sessions/sessions.md:1]: this is instead of `program.md`... it's fine, we can cleanup later

**Pull request merged**